### PR TITLE
2048 by Inlead: Adjust object fetching on userStatus.

### PR DESCRIPTION
**Pull request by NicolaeGrigoriu**
http://platform.dandigbib.org/issues/2048

### CODE REVIEW

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 63034daa5854e5815eb42bf39f0c2ef98ec5425a
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 63034daa5854e5815eb42bf39f0c2ef98ec5425a
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

File: modules/ding_provider/ding_provider.module [hunk 1]
```diff
@@ -612,144 +612,15 @@ function ding_provider_get_pseudo_entity($ding_entity_id) {
...
-
-    default:
-      // The agency ID was not translatable, do to one-to-may match from old pid
-      // to new pid. Or the agency ID is not known. So simply log it under a
-      // filterable type.
-      watchdog('ding_provider_pid', 'PID creation failed with agency (%agency) and ID (%id)', array('%agency' => $agency, '%id' => $id), WATCHDOG_ERROR);
-      break;
-  }
+function ding_provider_build_entity_id($id) {
+  $obj = ting_do_search($id . ' AND facet.acSource=bibliotekskatalog');
```
Comment (JrgenGNielsen) [inline modules/ding_provider/ding_provider.module]: should be: ting_do_search('rec.id=' . $id . ' AND facet.acSource=bibliotekskatalog');

**Comment by JrgenGNielsen** [comment]
reviewed and accepted

**Pull request merged**